use-fstring.yaml

```diff
@@ -0,0 +1,19 @@
+# This file contains optional rules
+# that spot when f-string should be used
+# instead of an older string formatting mechanism.
+#
+# Related default rules:
+#
+# https://docs.sourcery.ai/Reference/Default-Rules/refactorings/use-fstring-for-concatenation/
+# https://docs.sourcery.ai/Reference/Default-Rules/refactorings/use-fstring-for-formatting/
+
+rules:
+
+# You can use this rule in the Sourcery CLI with the --enable option:
+#
+# sourcery review --enable no-str-format .
+#
+- id: no-str-format
```

Comment:
slighlty worried about name-space clashes now that we have rules coming in from various different places - not entirely sure what to do about it.

Reply:
That's an excellent point. :+1: 

I've created a test PR #3 , and it seems our CI currently doesn't catch ID clashes.

Suggestion:

* Add a test to the internal CI that ensures that each ID is unique among the (default U optional) rules
* As long as this test doesn't exist: We should verify during the review that there's no ID clash. (I guess a text search for the ID in the internal repo is sufficient.)

Reply:
The name `no-str-format` seems to be available. :heavy_check_mark:

Reply:
Could you add such.a test? Would be better than the manual check.

Reply:
I've created an internal issue for it.

Reply:
Please add the test rather than waiting for the internal issue - should be fairly straightforward.